rosbot_description/urdf/wheel.urdf.xacro

```diff
@@ -61,9 +61,7 @@
 
       <inertial>
         <mass value="0.100" />
-        <inertia ixx="0.001" ixy="0.000" ixz="0.000"
-                             iyy="0.001" iyz="0.000"
-                                         izz="0.001" />
+        <inertia ixx="0.00007" ixy="0" ixz="0" iyy="0.0001263" iyz="0" izz="0.00007" />
```

Comment:
I prefer to type inertia as a matrix.

Reply:
My bad. This was not up to our standard. There are few more issues in regard to our standard, but I didn't want to touch them. Anyway fixed this one